### PR TITLE
Fix list doesnt display errors

### PR DESCRIPTION
**Pull request by lmr**
Fixes the card https://trello.com/c/WOPxkrZ8/450-bug-the-paginator-is-hiding-warnings-errors

It does that by fixing a View bug, and making the lister plugin to display plugin loading errors in case the paginator is enabled.

### CODE REVIEW

**Comment by ldoktor** [comment]
Well, this doesn't look like a proper fix to me. It only solves the problem with plugin loading errors, but other messages are again hidden by paginator.

For example let's break  gdb plugin's `activate`. When you run `avocado list` without paginator:

```
./scripts/avocado list
Could not activate plugin 'gdb': 
INSTRUMENTED /home/medic/Work/Projekty/avocado/avocado/examples/tests/datadir.py:DataDirTest.test
INSTRUMENTED /home/medic/Work/Projekty/avocado/avocado/examples/tests/skip_outside_setup.py:SkipOutsideSetup.test
```

With paginator:
```
INSTRUMENTED /home/medic/Work/Projekty/avocado/avocado/examples/tests/datadir.py:DataDirTest.test
INSTRUMENTED /home/medic/Work/Projekty/avocado/avocado/examples/tests/skip_outside_setup.py:SkipOutsideSetup.test
```

So we should find way to propagate all outputs to paginator.

**Comment by lmr** [comment]
@ldoktor, the loading message failure is done way earlier than the View object, therefore the paginator itself is initialized. This was a deliberate decision and was discussed previously, on the basis that the user needs to see the errors and all that jazz.

This should be a quick fix for the behavior that was reported. If we are looking for a 'more correct' fix, that fix would be probably:

1) Make test Views() a borg class
2) Initialize the View early when loading the plugins, then later on, the plugin would create another instance of View with shared state.

This should solve the problem. But there's a 'but'.

We can't currently switch on the fly whether we are using paginator and non paginator, and have the view class start with the paginator is a terrible default (we don't want paginators being used everywhere, specially in the runner itself).

So long story short, if we are going to fix this properly, i strongly suspect this is not a 'long hanging fruit' type of fix anymore. Or we could rip the paginator all across avocado, which I suspect is not what we want.

**Comment by lmr** [comment]
I'm back from lunch. Maybe we can do this by using a Borg class that's also lazily initialized. We could have a mechanism to queue up messages until the class actually has to be initialized, then show then in sequence.

**Comment by lmr** [comment]
I'm closing this since we agree this is not a good fix. We discussed some ideas, but they can't be properly implemented by today.
